docs/source/deploying-volttron/multi-platform/multi-platform-rabbitmq-deployment.rst

```diff
@@ -192,13 +171,26 @@ upstream servers on the downstream server and make the VOLTTRON exchange
 
         If no config file is provided, the script will prompt for
         hostname (or IP address), port, and vhost of each upstream node you
-        would like to add. Hostname provided should match the hostname in the
-        SSL certificate of the upstream server. For bi-directional data flow,
+        would like to add and certificates for connecting to upstream server. For bi-directional data flow,
         we will have to run the same script on both the nodes.
 
-    b.  Create a user in the upstream server (publisher) and provide it access to the virtual host of the upstream RabbitMQ server.
-        The username should be set to the instance name of the downstream server; it should take the form of <instance name>-admin.
-        For example, if the downstream server name is "volttron2", then the instance name would be "volttron2-admin".
+    b.  If no config file is provided and certificates for connecting to upstream server have to be generated afresh, then the upstream server should be web enabled and admin should be ready to accept/reject incoming requests. Please refer to :ref:`Multiple Platform Multiple Bus connection <Multi-Platform-Multi-Bus>` on how to enable web feature and accept/reject incoming authentication requests. Below image shows steps to follow to create a federation link from downstream instance "volttron1" to upstream instance "volttron2".
+
+        On downstream server (collector node),
+
+        .. image:: files/cmd_line_federation.png
+
+        On upstream server (publisher node), Login to "https://volttron2:8443/index.html" in a web browser. You will see incoming CSR request from "volttron1" instance.
+
+        .. image:: files/admin_request_federation.png
+
+        Accept the incoming CSR request from "volttron1" instance.
+
+        .. image:: files/csr_accepted_federation.png
+
+    c.  Create a user in the upstream server (publisher) and provide it access to the virtual host of the upstream RabbitMQ server.
```

Comment:
Do we still have to create the user manually? I thought this is not needed anymore

Reply:
Yes we need to. This is different from shovel.